### ISSUE TITLE
ES6 is no more

### ISSUE THREAD

**Issue by Kflash** [babel/babel-library-boilerplate#153]
Hi

I just open this issue to let you know that ES6 is not anymore, so you should change the names in this repo. The correct name now is **ES2015** It has been like this since january 2015.

You can google this and also check out the specs, but here is one nice thread to study:

https://esdiscuss.org/topic/javascript-2015

**Comment by jmeas** [comment]
> The correct name now is ES2015 It has been like this since january 2015.

Meh, people know what it means when you reference ES6, which is what's most important. Most people I know still call it ES6 colloquially, too.

In any case, I actually wonder if using ES2015 is also inaccurate in a different way. What I mean is that this is technically a *Babel* boilerplate, so it supports whatever Babel supports. O' course, Babel isn't restricted to any particular version of JavaScript...it's just future JavaScript.

I wonder if there's some way we could future-proof the README by coming up with language that communicates that, rather than sticking with a particular version.

**Comment by nicksrandall** [comment]
I like the term ESNext for what it's worth.

**Comment by Kflash** [comment]
EsNext was one of the name suggestions for the specs too.

I like it and it state clear what it is without messing with year and
version numbers as in es2015 and upcoming es2016

**Comment by jmeas** [comment]
I'm just gonna stick with ES6, I think. Using "ES6/2015" is too wordy to use everywhere, and the term [ES2015 doesn't have anywhere near as much recognition as ES6](https://www.google.com/trends/explore#q=es6%2C%20es2015&cmpt=q&tz=). There's not much gain in making the change, I think.

**Comment by thejameskyle** [comment]
Note: Babel always refers to it as ES6 and not ES2015. See the [homepage](http://babeljs.io) for a bunch of uses.

**Comment by jmeas** [comment]
Babel updated its homepage to use ES2015, so I'm going to switch the boilerplate over, as well.

**Comment by Kflash** [comment]
I don't care. You are an idiot anyway. Just stop spam me!!!

**Comment by jmeas** [comment]
> Just stop spam me!!!

Friend, you're the one subscribed to an issue in my repository. You can unsubscribe by clicking the button on the right-hand side of the page.

**Comment by Kflash** [comment]
@jmeas I updated my comment. Read again.
